db/migrate/20150130054629_add_category_to_assessments.rb

```diff
@@ -0,0 +1,18 @@
+class AddCategoryToAssessments < ActiveRecord::Migration
+  def change
+    add_column :assessments, :category_name, :string
+    
+    Assessment.all.each do |assessment|
+        sql = "SELECT name FROM assessment_categories WHERE id=" + assessment.category_id.to_s
+		records_array = ActiveRecord::Base.connection.execute(sql)
+	   	
+	   	records_array.each do |cat|
+    		assessment.category_name = cat[0]
+			assessment.save
+	    end
+    end
```

Comment:
Ah, well this should be fine, but yeah, I'm not sure we want to drop AssessmentCategories so quickly.  We may want to keep that table around for a bit until we're sure that grades aren't getting screwed up...